src/v2/Utils/Hooks/useLazyLoadComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react"
+import { LazyLoadComponent } from "react-lazy-load-image-component"
+
+const Ready = ({ onMount }) => {
+  useEffect(onMount, [onMount])
+  return null
+}
+
+/**
+ * Simple hooks wrapper around LazyLoadComponent.
+ * Allows one to swap out components when mounted.
+ * TODO: Replace this with a normal intersection observer hook.
```

Comment:
So internally this library defaults to an intersection observer. This is just a convoluted way of getting a hook interface so that we can swap out based on the boolean.